src/components/ConsentPicker.vue

```diff
@@ -0,0 +1,486 @@
+<template>
+  <PvPanel class="m-0 p-0 w-full" header="Select params for the Consent and Assent forms">
```

Comment:
Would we be able to change the title to `Select Consent and Assent Forms` without losing any of the original meaning? I feel that the briefer title might be more suitable.

Reply:
YES!